src/css/SpotifyUsername.css

```diff
@@ -1,23 +1,40 @@
 .rounded {

```

Comment:
this style can be splitted to 2: rounded (with border-radius) and profile-image (with w/h), then media queries can affect only profile-image style and rounded style will be just to make element 'rounded'